config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  namespace :api, defaults: { format: 'json' } do
+  namespace :api, defaults: { format: :json } do
```

Comment:
You should check the main branch because this change has been updated.

Reply:
rebase done. Thnaks!